src/components/era_commons/ERACommons.tsx

```diff
@@ -42,10 +33,10 @@ export default function ERACommons({
   required = false,
   destination = '',
   researcherProfile,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Follow on work to be addressed in https://broadworkbench.atlassian.net/browse/DT-2305